packages/react-native/src/primitives/Radio/Radio.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useMemo } from 'react';
+import { Pressable, View, ViewStyle } from 'react-native';
+
+import { Label } from '../index';
+
+import { RadioProps } from './types';
+import { styles } from './styles';
+
+export default function Radio<T>({
+  buttonStyle,
+  disabled,
+  label,
+  labelPosition = 'end',
+  labelStyle,
+  onChange,
+  selected,
+  size,
+  style,
+  value,
+  ...rest
+}: RadioProps<T>): JSX.Element {
+  const handleOnChange = useCallback(() => {
+    if (!disabled) {
+      onChange?.(value);
+    }
+  }, [onChange, value, disabled]);
+
+  const containerDirection: ViewStyle = useMemo(
+    () => ({
+      flexDirection:
+        labelPosition === 'bottom' || labelPosition === 'top'
+          ? 'column'
+          : 'row',
+    }),
+    [labelPosition]
+  );
+
+  const labelPrecedesButton =
+    labelPosition === 'start' || labelPosition === 'top';
+
+  return (
+    <View
```

Comment:
If we follow the UX of our React web `RadioGroupField` implementation, the entire `Radio` element should be handle press, so let's replace this `View` with a `Pressable`